### ISSUE TITLE
New blocks seem not to be received in background

### ISSUE THREAD

**Issue by mmazi** [ksedgwic/Wallet32#3]
After several confirmations in the blockchain, I was still unable to send coins and the receiving transaction status was still P. After exiting and restarting the application, new blocks were downloaded and then I could do it.

![screenshot_2013-12-27-02-52-29](https://f.cloud.github.com/assets/294075/1814305/3b509354-6ee8-11e3-8f35-702c0f4ef8dd.png)

**Comment by ksedgwic** [comment]
There are two things that come to mind:

1) I need to upgrade to a more recent version of bitcoinj; it is possible that this is bug that's already been fixed.

2) I need to add a way to export the logs in a useful way, this would allow further investigation.

**Comment by ksedgwic** [comment]
I've seen two cases of this since.

In the first case both the sending and receiving wallets saw the transaction, but the sending wallet did not see the first (or any additional) confirmation.

In the second case it was the receiving wallet that saw the transaction but not the confirmation.

In both cases restarting Wallet32 immediately cleared up the problem.

Working on getting logs still, running into Android DEX file limitations ... sigh ...

**Comment by ksedgwic** [comment]
Matija, I think this is fixed in version-0.1.7 with the upgrade to bitcoinj-0.11-SNAPSHOT.

Could you retest again?

Thanks!

Ken

**Comment by mmazi** [comment]
I have 0.1.7 and the problem seems to persist. Wallet32 received a transaction which is still shown as P, even though there have been 3 confirmations in the network by now.

**Comment by mmazi** [comment]
Actually, later on the status of the transaction in Wallet32 changed; I also retested this a couple of times and it seems fine now. Sorry for the false alert.

TL;DR I think this is fine now.
